src/services/order/order.service.ts

```diff
@@ -3,12 +3,15 @@ import { User, UserAddress } from '@entities/auth';
 import { Cart } from '@entities/cart';
 import { Order, OrderItem } from '@entities/order';
 import { PaymentStatus } from '@enums/order';
+import { PaymentIntentEvent } from '@enums/stripe';
 import { ConflictException, Logger, NotFoundException } from '@nestjs/common';
 import { InjectRepository } from '@nestjs/typeorm';
 import { UserRepository } from '@repository/auth';
 import { CartRepository } from '@repository/cart';
 import { OrderRepository } from '@repository/order';
 import { ProductService } from '@services/product';
+import { StripeService } from '@services/stripe';
+import { Stripe } from 'stripe';
```

Comment:
Here, if you're only going to use a specific type `Stripe.Event` would be better to import just that and not the whole thing.